src/intelligence_layer/examples/qa/multiple_chunk_qa.py

```diff
@@ -29,11 +29,13 @@ class MultipleChunkQaInput(BaseModel):
             Can be arbitrarily long list of chunks.
         question: The question that will be answered based on the chunks.
         language: The desired language of the answer. ISO 619 str with language e.g. en, fr, etc.
+        generate_highlights: Whether to generate highlights (using the explainability feature) for the answer.  Defaults to `False` for performance reasons.
```

Comment:
Docstring default no longer matches the actual default for all the docstrings.

Reply:
Correct. Thanks for catching this! Fixed here - https://github.com/Aleph-Alpha/intelligence-layer-sdk/pull/1018/commits/17da13fb82e3c460c377bf76a8cbfa5eb7173e7b